src/base/datetime/date/index.ts

```diff
@@ -27,14 +30,25 @@ export class BaseDate extends KucBase {
   @state()
   private _dateTimeCalendarVisible = false;
   private _locale = getLocale("en");
-
+  private _calendarValue?: string = "";
   update(changedProperties: PropertyValues) {
     if (changedProperties.has("inputId")) {
       this._GUID = this.inputId;
     }
     if (changedProperties.has("language")) {
       this._locale = getLocale(this.language);
     }
+    if (changedProperties.has("value")) {
+      if (
+        isStringValueEmpty(this.value) ||
+        !isValidDateFormat(this.value, Common_Language)
```

Comment:
@h000780 
Why do we need to declare `Common_Language`?
Can we using `this.language` for this case?